src/Apps/W1/Subscription Billing/App/Contract Renewal/Tables/PlannedSubscriptionLine.Table.al

```diff
@@ -206,6 +207,8 @@ table 8002 "Planned Subscription Line"
             begin
                 DateFormulaManagement.ErrorIfDateFormulaEmpty("Billing Rhythm", FieldCaption("Billing Rhythm"));
                 DateFormulaManagement.ErrorIfDateFormulaNegative("Billing Rhythm");
+                if (Format("Billing Base Period") <> '') and (Format("Billing Rhythm") <> '') then
```

Comment:
This is already checked inside `CheckRatioBetweenBillingBasePeriodAndRhythm()`:
```suggestion
```

Reply:
That was my oversight obviously